### DISCUSSION TITLE
[Feature Request] Versioning of Alert Rules

### DISCUSSION THREAD

**Discussion by BlackDex** [grafana/grafana#73152] [Alerting]
Hello there,

Before Unified Alerting all the alert rules were stored in the dashboard json.
With every change this Alert Rule was also stored in the versioning of the dashboard json, and thus in case of an issue you were able to at least lookup the older version of the alert.

It would be super handy if there also is a versioning for Alert Rules, at least for the Rules edited via the UI.
For provisioned Alert Rules it probably isn't needed or wanted, since that should be stored elsewhere.

What do you think about this?

Cheers.

**Comment by gelicia** [comment]
Please see #73936 - thank you!
